protected/humhub/components/Migration.php

```diff
@@ -18,6 +18,58 @@
  */
 class Migration extends \yii\db\Migration
 {
+    public const LOG_CATEGORY = 'migration';
+
+    /**
+     * @inheritdoc
+     * @author Martin Rüegg
```

Comment:
You are welcome to add your name as an author in the class header.  At method level we do not use the '@author` tag.

Reply:
Ok, there have been many authors on this file. Either I add all who significantly contributed, or I don't add myself either ... ;-)

Reply:
Ok, I understand, I would prefer not to include the authors in the PHPDoc again, since they are collected via Git & GitHub anyway. 

Also on method level PHPStorm shows the authors, e.g.
![image](https://user-images.githubusercontent.com/4736168/232462628-22faffe7-4b46-4c3d-9e8f-6ac9e6857625.png)

But if you want, feel free to update the class header with everyone or you as the author.